src/utils/createTestAttr.ts

```diff
@@ -0,0 +1,6 @@
+const dataAttrName = 'testid';
+
+export function createTestAttr(id: string, type: 'data-' | 'id' = 'data-') {
+  if (type === 'id') return { id };
+  return { [`data-${dataAttrName}`]: id };
+}
```

Comment:
I think it's better to use data attr in general ( what I mean is that we should use one way )
so in that case I don't think we need this function, if we need it we should also make use of it in the test files 

and the reason I don't agree with having ids ( which if u remember I was actually agreeing with before ) is that some 3rd party design lib ( like mantine ) makes use of that id field so we need something really specific than using a known attr.

Reply:
yes, it's better to use data attr. ID is an option.
At first, I created another function to make selectors for our test like`'[data-testid="something"]` but then I found out that `cypress-testing-library` has one which we can change the default `data-testid` so I removed mine.

but I've created this function for two reaseon:
1. use the same IDs in my test (without writing them again in my test file)
2. to prevent typos: In one of my tests, I wrote my data attr like this `<Input data-**testdi**="something"/>` . so I had a failing test just for a typo in my data-attr